test/functional/test_site/expected/index.html

```diff
@@ -183,6 +183,16 @@ <h1 id="page-variable-referencing-included-variable">Page Variable Referencing I
             <div></div>
             Page Variable Referencing Included Variable
           </div>
+          <div>
+            <h1 id="imported-variable">Imported Variable<a class="fa fa-anchor" href="#imported-variable"></a></h1>
+            <div></div>
```

Comment:
The normal text "Imported Variable" did not appear here so this test case failed?

Reply:
Ah my bad, I think it might've disappeared when I was moving stuff around